src/api/config/initializers/flipper.rb

```diff
@@ -15,21 +14,17 @@
   end
 
   Flipper.register(:beta) do |user|
-    user.respond_to?(:in_beta?) && user.in_beta?
+    # The user has to be in beta for this group to be active...
+    user.respond_to?(:in_beta?) && user.in_beta? &&
+      # ...and if the user didn't disable the feature, it will be active
+      user.respond_to?(:disabled_beta_features) && !user.disabled_beta_features.exists?(name: feature_toggle_name)
```

Comment:
`feature_toggle_name` doesn't exist in this context. On top of this, the implementation in this PR doesn't work since it's a all-or-nothing approach like we had before for the beta program. We need to have a Flipper group for every feature toggle part of the beta program. Without this, there is no granularity and users of the beta program cannot enable/disable a specific beta feature.